Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -2501,6 +2523,143 @@ def get_runtime_container_audit_events(client: PrismaCloudComputeClient, args: d
     )
 
 
+def archive_audit_incident_command(client: PrismaCloudComputeClient, args: dict) -> str:
+    """
+        Archives or Unarchives the audit incident according to the provided incident ID
+
+        Args:
+            client (PrismaCloudComputeClient): prisma-cloud-compute client.
+            args (dict):prisma-cloud-archive-audit-incident command arguments.
+
+        Returns:
+            string: A string that indicates success or failure
+    """
+    incident_id = args.get("incident_id") or ""
+    if args.get("action") == "archive":
+        data = '{"acknowledged": true}'
+    else:
+        data = '{"acknowledged": false}'
+
+    client.archive_audit_incident(incident_id=incident_id, data=data)
+    if "true" in data:
+        return f'Incident {incident_id} was successfully archived'
+    else:
+        return f'Incident {incident_id} was successfully unarchived'
```

Comment:
The API does not return any other info in the response?
```suggestion
    return f'Incident {incident_id} was successfully  {"archived" if args.get("action") == "archive" else "unarchived"}'
```